src/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,71 @@
+import { ComponentProps } from 'react';
+import { getCommonProps } from '../../utils';
+import classnames from 'classnames';
+import * as Tabs from '@radix-ui/react-tabs';
+
+// Define the type for each tab, which includes its label, value, and content
+export interface Tab {
+  /**
+   * button label
+   */
+  label: string;
+  /**
+   * Button value
+   */
+  value: string;
+  /**
+   * Content to display on tab selection
+   */
+  content?: React.ReactNode;
+}
+
+/**
+ * TabsComponent renders a tab interface allowing switching between different content.
+ *
+ * @param {TabsComponentProps} props - The component props containing an array of tabs.
+ * @returns {JSX.Element} The rendered tab component.
+ */
+export interface TabsComponentProps extends ComponentProps<'div'> {
+  tabs: Tab[];
+  /**
+   * Specify the default tab
+   */
+  defaultValue?: string;
+}
+
+/**
+ * ## Overview
+ *
+ * Overview of Tabs component
+ *
+ * [Figma Link](https://www.figma.com/file/hMu9IWH5N3KamJy8tLFdyV?type=design&node-id=10838%3A22184&mode=dev)
+ *
+ * [Storybook Link](https://phillips-seldon.netlify.app/?path=/docs/components-tabs--overview)
+ */
+
+const TabsComponent = ({ className, tabs = [], defaultValue, ...props }: TabsComponentProps) => {
```

Comment:
could you wrap this component in `forwardRef` to this component as well as any of the composable components we come up with

Reply:
you'll typically get this for free if you use the create-component script:
https://github.com/PhillipsAuctionHouse/seldon/blob/2a1de144ec0be4a7d34c4d0745847030ad7cad00/.template/Component/ComponentName.tsx#L16